hazelcast-tpc-engine/src/main/java/com/hazelcast/internal/tpc/iobuffer/ConcurrentIOBufferAllocator.java

```diff
@@ -29,13 +29,17 @@
  * It also allows the {@link #allocate()} of a {@link IOBuffer} to be done by a different
  * thread than {@link #free(IOBuffer)}.
  */
+@SuppressWarnings("checkstyle:MagicNumber")
 public class ConcurrentIOBufferAllocator implements IOBufferAllocator {
 
-    private static final AtomicLong newAllocations = new AtomicLong(0);
-    private static final AtomicLong pooledAllocations = new AtomicLong(0);
-    private static final AtomicLong allocateCalls = new AtomicLong();
-    private static final AtomicLong releaseCalls = new AtomicLong();
-    private final static ThreadLocal<Pool> POOL = new ThreadLocal<>();
+    // this is for debugging. We do not want to track these calls normally
+    private static final AtomicLong NEW_ALLOCATIONS = new AtomicLong(0);
+    private static final AtomicLong POOLED_ALLOCATIONS = new AtomicLong(0);
```

Comment:
```suggestion
    private static final AtomicLong NEW_ALLOCATIONS = new AtomicLong();
    private static final AtomicLong POOLED_ALLOCATIONS = new AtomicLong();
```
minor: just to be consistent with other two

Reply:
Fixed